convex/subscriptions.ts

```
@@ -306,7 +307,7 @@ export const manageTeamOnSubscriptionChange = internalMutation({
       // Delete the team
       const team = await ctx.db
         .query('teams')
-        .filter((q) => q.eq(q.field('id'), oldSubscription.team_id!))
+        .withIndex('by_team_id', (q) => q.eq('id', oldSubscription.team_id!))
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)